frontend/spec/features/products_filtering_spec.rb

```diff
@@ -73,4 +81,27 @@ def filters
       expect(filters).not_to have_content('Color')
     end
   end
+
+  context 'property filters' do
```

Comment:
@aplegatt we also need tests to test that those filters are actually in use... I don't see anywhere in https://github.com/spree/spree/blob/master/core/app/finders/spree/products/find.rb filtering by property